website/data/plugins-nav-data.json

```
@@ -0,0 +1,8 @@
+[
+  {
+    "heading": "About External Plugins"
+  },
```

Another loosely held hot take, but I think it might be more helpful to link to the overview page here rather than have a static heading

```suggestion
  {
    "title": "About External Plugins",
    "path": ""
  },
```